tools/mage/translations.js

```diff
@@ -200,7 +204,7 @@ const readMessages = async function () {
     }, {})
 }
 
-/**
+/** .........
```

Comment:
I still see the dots - are they intentional ?

Reply:
Hmm, seems to be a bug with my formatter. I'll check again.

Reply:
Now gone.